internal/gl/gl_js.js

```diff
@@ -0,0 +1,18 @@
+(() => {
+
+    window._fastReadPixels = function (fn, x, y, width, height, format, ty, dataPtr, dataLen) {
+        fn(x, y, width, height, format, ty, new Uint8Array(go._inst.exports.mem.buffer, dataPtr, dataLen))

```

Comment:
`go._inst.exports.mem.buffer` is internal to the Go runtime, no?

Reply:
That is "public", on JS perspective, since it's exported. But, I think it's not worth. The main issue is the `makeArgs` from `syscall/js` and the `js.Value` which causes alot of allocations. Removing the `CopyBytesToJS` and `New` will safe some allocs, but not significant.

I'm closing it. I think that is the best that we can do with `syscall/js`.